annotator/src/main/java/org/cbioportal/models/MutationRecord.java

```diff
@@ -507,4 +509,61 @@ public void initHeader() {
         header.add("n_ref_count");
         header.add("n_alt_count");
     }
+
+    private static List<String> initMissingValueList() {
+        List<String> missingValueList = new ArrayList<>();
+        missingValueList.add("NA");
+        missingValueList.add("N/A");
+        missingValueList.add("N/a");
+        missingValueList.add("n/A");
+        missingValueList.add("Unknown");
+        missingValueList.add("not available");
+        return missingValueList;
+    }
+
+    public static boolean hasMissingKeys(String check) {
+        for (String ignore : missingValueList) {
+            if (check.equalsIgnoreCase(ignore)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Note to self: Only comparing fields that define a mutation here - not every field is checked.